var/spack/repos/builtin/packages/libpsm3/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Libpsm3(AutotoolsPackage):
+    """PSM3 provider for the PSM3 OFI provider"""
+
+    homepage = "https://ofiwg.github.io/libfabric/v1.12.0/man/fi_psm3.7.html"
+    url = "https://github.com/intel/eth-psm3-fi/archive/refs/tags/v11.4.1.0.tar.gz"
+    git = "https://github.com/intel/eth-psm3-fi.git"
+
+    version(
+        "11.4.1.0",
+        sha256="272adb9ec10edf709bfcfccc6b6e9296d25d892c36b845ad577caeb82b70c9ac",
+        preferred=True,
+    )
+
+    variant("verbs", default=False, description="Enable PSM3 verbs")
+
+    variant("sockets", default=True, description="Enable PSM3 sockets")
+
+    variant("atomics", default=True, description="Enable atomics")
+
+    variant("debug", default=False, description="Enable debugging")
```

Comment:
I recommending alphabetizing these by variant name for maintainability.

Reply:
Done, see latest commit